containerd-shim-wws-v1/Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG CROSS_BASE_IMAGE
+FROM $CROSS_BASE_IMAGE
+
+# ENV CC="x86_64-linux-musl-gcc"
```

Comment:
It looks like all the envs are commented out. Do we still need this Dockerfile?

Reply:
Oh! This is a leftover from my local development setup. I need to figure out why I need these specific environment variables in my local environmnet, but it shouldn't be required on the CI or other environments.